src/backend/storeManagers/legendary/library.ts

```diff
@@ -744,6 +745,57 @@ export async function getGameSdl(
 }
 
 /**
+ * Toggles the EGL synchronization on/off based on arguments
+ * @param path_or_action On Windows: "unlink" (turn off), "windows" (turn on). On linux/mac: "unlink" (turn off), any other string (prefix path)
+ * @returns string with stdout + stderr, or error message
+ */
+export async function toggleGamesSync(path_or_action: string) {
+  const isWindows = platform() === 'win32'
+
+  if (isWindows) {
+    const egl_manifestPath =
+      'C:\\ProgramData\\Epic\\EpicGamesLauncher\\Data\\Manifests'
+
+    if (!existsSync(egl_manifestPath)) {
+      mkdirSync(egl_manifestPath, { recursive: true })
+    }
+  }
+
+  const command: LegendaryCommand = {
+    subcommand: 'egl-sync',
+    '-y': true
+  }
+
+  if (path_or_action === 'unlink') {
+    command['--unlink'] = true
+  } else {
+    command['--enable-sync'] = true
+    if (!isWindows) {
+      command['--egl-wine-prefix'] = Path.parse(path_or_action)
```

Comment:
I don't think we can be certain that this path is actually *valid* (it's user input after all; trusting the OS's file picker to do the validation for us sounds like a bad idea). Might want to do something like
```suggestion
      const pathParse = Path.safeParse(path_or_action)
      if (pathParse.success)
        command['--egl-wine-prefix'] = pathParse.data
      else
        return 'Error'
```

Doing this at the source would be the long-term goal of course, but that's far too great of a task to just push onto this PR :^)